SeaBattle/Sea.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SeaBattle
+{
+    public class Sea
+    {
+        private List<Deck> Decks { get; set; }
+        private List<BaseShip> Ships { get; set; }
+        int SeaWidth;
+        int SeaLenght;
+
+        public Sea()
+        {
+            var Ships = new List<BaseShip>();
+        }
+
+
+        public void Create()
+        {
+            //will finish this IF parts tommorow. 
+            //if ( > 10 &&  > 10)
+            //{
+            //}
+
+            for (int i = 0; i < Ships.Count; i++)
+            {
+                for (int j = 0; j < Decks.Count; j++)
+                {
+
+                }
+
+            }
+
+            var items = new List<PointStruct>(4);
+
+            items.Add(new PointStruct() { X = 2, Y = 1 });
+            items.Add(new PointStruct() { X = 2, Y = 2 });
+            items.Add(new PointStruct() { X = 2, Y = 3 });
+            items.Add(new PointStruct() { X = 2, Y = 4 });
+
+            var zxc = new List<PointStruct>(2);
+
+            zxc.Add(new PointStruct() { X = 6, Y = 2 });
+            zxc.Add(new PointStruct() { X = 3, Y = 3 });
+
+            var RepairerShip = new List<PointStruct>(3);
+
+            RepairerShip.Add(new PointStruct() { X = 6, Y = 7 });
+            RepairerShip.Add(new PointStruct() { X = 6, Y = 8 });
+            RepairerShip.Add(new PointStruct() { X = 6, Y = 9 });
+
+            var ImAHealerShip = new List<PointStruct>(3);
+
+            ImAHealerShip.Add(new PointStruct() { X = 8, Y = 7 });
+            ImAHealerShip.Add(new PointStruct() { X = 8, Y = 8 });
+            ImAHealerShip.Add(new PointStruct() { X = 8, Y = 9 });
+
+            var Hybrydo = new List<PointStruct>(3);
+
+            Hybrydo.Add(new PointStruct() { X = 2, Y = 5 });
+            Hybrydo.Add(new PointStruct() { X = 3, Y = 5 });
+            Hybrydo.Add(new PointStruct() { X = 4, Y = 5 });
+
+            var tendari = new List<PointStruct>(3);
+
+            tendari.Add(new PointStruct() { X = 3, Y = 9 });
+            tendari.Add(new PointStruct() { X = 2, Y = 9 });
+            tendari.Add(new PointStruct() { X = 1, Y = 9 });
+
+            //I am creating 1  x4deck and 1 x2deck BattleShip, 2 x3deck Serviceship, 1 x2deck and x4deck ServiceShip.
+            BattleShip ship1 = new BattleShip("Yamato", items);
```

Comment:
Why do you create ships inside the Create() method? How do you gonna use them?

Reply:
It`s a good question, I think I lost method AddShip() which will add ships on the board.